xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -202,7 +202,7 @@ bool CGUIWindowVideoBase::OnMessage(CGUIMessage& message)
 void CGUIWindowVideoBase::OnItemInfo(const CFileItem& fileItem, ADDON::ScraperPtr& scraper)
 {
   if (fileItem.IsParentFolder() || fileItem.m_bIsShareOrDrive || fileItem.IsPath("add") ||
-     (fileItem.IsPlayList() && !URIUtils::HasExtension(fileItem.GetPath(), ".strm")))
+  	 (fileItem.IsPlayList() && !URIUtils::HasExtension(fileItem.GetDynPath(), ".strm")))
```

Comment:
IsPlaylist already uses GetDynPath. I think we can drop the extra check for strm here.

Reply:
@FernetMenta I assume this check is to continue OnItemInfo processing (and show info dialog) when item is a .strm playlist, but not any other kind of playlist

Reply:
Thank you for your reply. That was my initial thought. I removed the extra check, but it didn't fix the issue when I tested. I assume it is because strm is considered a playlist, right? That's why there's the extra check for strm, so the information dialog is displayed even if it's a strm file.

Reply:
Yes @angelblue05 , IsPlaylist() is true for .strm files, see
https://github.com/xbmc/xbmc/blob/33700e0b57beaf512567b754db9e989208b53690/xbmc/playlists/PlayListFactory.cpp#L145

Reply:
But to nit pick the white space needs correction in both places

Reply:
Ok, I will fix the white spaces. @DaveTBlake